content/zh/docs/demos/canary_rollout.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "使用 SMI 流量拆分进行金丝雀部署"
+description: "使用 SMI 流量拆分管理金丝雀部署"
+type: docs
+weight: 21
+---
+
+本指南演示如何使用 SMI 流量拆分配置执行金丝雀部署。
+
+## 先决条件
+
+- Kubernetes 集群版本 {{< param min_k8s_version >}} 或者更高。
+- 已安装 OSM。
+- 使用 `kubectl` 与 API server 交互。
+- 已安装 `osm`  命令行工具，用于管理服务网格。
+
+
+## 演示
+
+在此演示中，我们将部署一个 HTTP 应用程序并执行金丝雀发布，其中会部署新版本的应用程序，以对一定百分比的流量提供服务。
+
+为了拆分流量到多个服务后端，需要使用 [SMI 流量拆分 API](https://github.com/servicemeshinterface/smi-spec/blob/main/apis/traffic-split/v1alpha2/traffic-split.md) 。关于 API 使用的更多说明，可以查看 [流量拆分指南](/docs/guides/traffic_management/traffic_split)。为了将客户端流量透明地拆分到多个服务后端，需要注意客户端需要使用 `TrafficSplit` 资源中指定的根服务的 FQDN 来发送流量。在该演示中，`curl` 客户端发送流量到根服务 `httpbin.org`，该服务初始有 `v1` 版本的服务，然后执行金丝雀发布将部分流量定向到 `v2` 版本的服务中。
```

Comment:
> [流量拆分指南](/docs/guides/traffic_management/traffic_split)

This will redirect to the English version. Do we have a solution to link it to the Chinese version?